src/aics-image-viewer/shared/utils/controlPointsToLut.ts

```diff
@@ -216,16 +216,3 @@ export function rampToControlPoints([min, max]: [number, number]): ControlPoint[
     { x: Math.max(max + 1, TFEDITOR_MAX_BIN), opacity: 1, color: TFEDITOR_DEFAULT_COLOR },
   ];
 }
-
-/** Remaps an array of control points from an old range (as a 2-tuple) to a new one (extracted from a `Channel`) */
-export function remapControlPointsForChannel(
```

Comment:
No longer used

Reply:
is the vole-core remapControlPoints dead now too?